samples/RefreshFlow/AngularApp/karma.conf.js

```diff
@@ -0,0 +1,43 @@
+// Karma configuration file, see link for more information
```

Comment:
Are there tests for this sample project? If not, delete this test configuration file.